### ISSUE TITLE
pelias import all  -- gives errors on whosonfirst csv's and warning on wof service

### ISSUE THREAD

**Issue by mohammedayub44** [pelias/docker#106]
Hi,

Trying to run the sample docker example on Portland data. I have not changed anything but the data dir in .env file. All my previous commands have ran without any errors, but I get these errors on elasticsearch while it's trying to import certain whosonfirst datasets specifically - `whosonfirst-data-neighbourhood-latest.csv` and `whosonfirst-data-postalcode-latest.csv` 

![image](https://user-images.githubusercontent.com/7122670/58534499-7c8c1980-81b9-11e9-8904-e3f51476c253.png)

But then also it says import is done and does not show any `failed_records`  and also shows some unable to locate errors on 
![image](https://user-images.githubusercontent.com/7122670/58534895-777b9a00-81ba-11e9-86aa-5d2ebd6fac97.png)

Basically is this expected to happen and can I safely ignore them or something is missing.
@missinglink @orangejulius 

Appreciate any help !

**Comment by mohammedayub44** [comment]
Also when I run `pelias test run` it gives `FATAL_ERROR 45 Regressions detected` 

![image](https://user-images.githubusercontent.com/7122670/58535432-ff15d880-81bb-11e9-999c-1212bed855d6.png)

I see couple of them like these errors in the output:

![image](https://user-images.githubusercontent.com/7122670/58535517-33899480-81bc-11e9-97d9-760bc7d8e7e0.png)

**Comment by orangejulius** [comment]
Hi @mohammedayub44,
The test results showing a few failures is not uncommon, it often changes as data changes. The warnings are also expected, as there aren't any records in, for example, the `borough` layer for Portland.

I've also seen the Parse Error message a few times, but like you said it always goes away and does not seem to affect the import. If you or anyone else has a chance to dig into it and see what the underlying error is, it would be appreciated.

**Comment by mohammedayub44** [comment]
@orangejulius  Sounds good. For now I think this helps. I was able to run it successfully.
Will dig into it later for errors.
Closing as my original question was answered. 

Thanks for the help !
